instructions/bundling-scripts/zimbra-core.sh

```diff
@@ -359,6 +359,7 @@ main()
    Copy ${repoDir}/zm-core-utils/src/libexec/zmrrdfetch                                             ${repoDir}/zm-build/${currentPackage}/opt/zimbra/libexec/zmrrdfetch
    Copy ${repoDir}/zm-core-utils/src/libexec/zmsacompile                                            ${repoDir}/zm-build/${currentPackage}/opt/zimbra/libexec/zmsacompile
    Copy ${repoDir}/zm-core-utils/src/libexec/zmsaupdate                                             ${repoDir}/zm-build/${currentPackage}/opt/zimbra/libexec/zmsaupdate
+   Copy ${repoDir}/zm-core-utils/src/libexec/zmsearchNdelete                                        ${repoDir}/zm-build/${currentPackage}/opt/zimbra/libexec/zmsearchNdelete
```

Comment:
If we're "productizing" this utility, I think we may want to consider not using mixed case in the utility name
- zmsearchNdelete → zmsearchndelete

If we do that we should update the documentation in the utility. We could also:
1. update the license block
2. change all uses of '&' in the documentation with the word 'and'
3. Rewrite/reword the BUGS/KNOWN issues section. It doesn't flow very well and could use some cleanup.

Up to you if you choose to do any of these things though.